src/helpers/types.ts

```diff
@@ -0,0 +1,49 @@
+export type Module = {
+  id: string,
+  name: string,
+  url: string,
+
+  isThesis: boolean,
```

Comment:
What is `isThesis` used for?

Reply:
I added all available properties, but there are some, which we don't (yet) use. I'll remove those.

Reply:
Nah leave them, I was just wondering

Reply:
Now that you mentioned it, it would be much cleaner to only have "useful" properties. So I think I'll leave them out, we can always add them back in once they are needed.